R/dataframe-frame.R

```diff
@@ -201,22 +379,86 @@ dataframe__tail <- function(n = 5) {
   })
 }
 
+#' Drop columns of a DataFrame
+#'
+#' @param ... Characters of column names to drop. Passed to [`pl$col()`][pl_col].
+#' @param strict Validate that all column names exist in the schema and throw an
+#' exception if a column name does not exist in the schema.
+#'
+#' @inherit as_polars_df return
+#' @examples
+#' as_polars_df(mtcars)$drop(c("mpg", "hp"))
+#'
+#' # equivalent
+#' as_polars_df(mtcars)$drop("mpg", "hp")
 dataframe__drop <- function(..., strict = TRUE) {
   self$lazy()$drop(..., strict = strict)$collect(`_eager` = TRUE) |>
     wrap()
 }
 
 # TODO: accept formulas for type mapping
+#' Cast DataFrame column(s) to the specified dtype
+#'
+#' @inherit LazyFrame_cast description params
+#'
+#' @inherit as_polars_df return
+#' @examples
+#' df <- pl$DataFrame(
+#'   foo = 1:3,
+#'   bar = c(6, 7, 8),
+#'   ham = as.Date(c("2020-01-02", "2020-03-04", "2020-05-06"))
```

Comment:
```suggestion
#'   ham = as.Date(c("2020-01-02", "2020-03-04", "2020-05-06")),
```

Reply:
Why? I really don't want to change this by hand across all docs

Reply:
For consistency with the Python examples, but certainly not of great importance. Sorry.